internal/server/handler_v1.go

```diff
@@ -220,7 +220,20 @@ func (s *Server) BioPage(
 func (s *Server) PlacePage(
 	ctx context.Context, in *pbv1.PlacePageRequest,
 ) (*pbv1.PlacePageResponse, error) {
-	return page.PlacePage(ctx, in, s.store)
+	localResp, err := page.PlacePage(ctx, in, s.store)
+	if err != nil {
+		return nil, err
+	}
+	if localResp == nil && s.metadata.RemoteMixerDomain != "" {
```

Comment:
maybe check localResp.StatVarSeries here instead of setting localResp to nil?

Reply:
done